draft-gpew-priv-ppm.md

```diff
@@ -568,11 +569,11 @@ context and `context` is the HPKE context used by the client for encryption.
 The payload is encrypted as
 
 ~~~
-payload = context.Seal(time || nonce || extensions, input_share)
+payload = context.Seal(nonce || extensions, input_share)
```

Comment:
Do we need to spell out how a `struct Nonce` gets mashed into an HPKE context? i.e. endianness

Reply:
I don't think so. The HPKE function takes a blob and the TLS language specifies a bigendian serialization.

Reply:
TLS-syntax specifies the byte encoding of `struct Nonce`, including endianness. See also my comment here: https://github.com/abetterinternet/ppm-specification/issues/139#issuecomment-1002795321. (If I've understood you correctly, we should be able to close that issue without action.)